README.md

```diff
@@ -1,30 +1,44 @@
-# Hydrogen template: Skeleton
+# Hydrogen with Sentry
 
-Hydrogen is Shopify’s stack for headless commerce. Hydrogen is designed to dovetail with [Remix](https://remix.run/), Shopify’s full stack web framework. This template contains a **minimal setup** of components, queries and tooling to get started with Hydrogen.
+This is a demo for getting started with Hydrogen and Sentry. It's based on the Hydrogen's [default template](https://github.com/Shopify/hydrogen/tree/main/templates/skeleton) with TypeScript.
 
-[Check out Hydrogen docs](https://shopify.dev/custom-storefronts/hydrogen)
-[Get familiar with Remix](https://remix.run/docs/en/v1)
 
-## What's included
+[Sentry Remix SDK Docs](https://docs.sentry.io/platforms/javascript/guides/remix/)
 
-- Remix
-- Hydrogen
-- Oxygen
-- Shopify CLI
-- ESLint
-- Prettier
-- GraphQL generator
-- TypeScript and JavaScript flavors
-- Minimal setup of components and routes
+[Hydrogen Docs](https://shopify.dev/custom-storefronts/hydrogen)
+
+[Remix Docs](https://remix.run/docs/en/main)
 
 ## Getting started
 
 **Requirements:**
 
 - Node.js version 16.14.0 or higher
 
+**Setup:**
+
+1. `npm install`
+2. Place your DSN in `entry.client.tsx` and `entry.server.tsx`
```

Comment:
Should we populate DSN from environmental variable as well?

Reply:
Just tried it, and interestingly it seems the env variables in Hydrogen apps are not directly accessible from `process` (which is not defined). They are accessible inside loaders, actions and request handlers. We can't access them directly from the root of `entry.server.tsx`. https://github.com/Shopify/hydrogen/issues/714

There's probably a workaround but IMHO including that might make this demo look more complicated than it is.

Reply:
alright, let's not worry about it then.